.github/workflows/migration-tests.yml

```diff
@@ -265,7 +265,9 @@ jobs:
           terraform_wrapper: false    
       - name: Migration Tests
         env:
-          MONGODB_ATLAS_ENABLE_BETA: ${{ vars.MONGODB_ATLAS_ENABLE_BETA }}
+          MONGODB_ATLAS_ENABLE_PREVIEW: "true"
```

Comment:
knitpit: I think "true" and true  are equivalent here as env vars are passed as strings